### PR TITLE
Release v0.27.0

### PR DESCRIPTION
**Pull request by Farhad-Shabani**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,16 @@
 # CHANGELOG
 
+## v0.26.1
+
+*January 16, 2023*
+
+This release contains a bug fix for the `ChanOpenConfirm` handler and strongly recommended to upgrade.
+
```
Comment (plafer) [inline CHANGELOG.md:10]: Can you also add a statement that this is a consensus-breaking change in the channel handshake?

```
This release contains a consensus-breaking change during the channel opening handshake; it was broken, and now is fixed.
```

**Review by plafer — COMMENTED** [review]
Since this is a consensus-breaking change, we should make this a minor release (i.e. `0.27.0`)

File: CHANGELOG.md (hunk 1 above)
Comment (Farhad-Shabani) [reply CHANGELOG.md:10]: 🙏  Fixed

Comment (plafer) [reply CHANGELOG.md:10]: Actually we should have both statements. It should read:

```
*January 16, 2023*

This release contains a bug fix for the `ChanOpenConfirm` handler and it is strongly recommended to upgrade.

The change is consensus-breaking (during the channel opening handshake); it was broken, and now is fixed.
```

Comment (Farhad-Shabani) [reply CHANGELOG.md:10]: My bad... done

**Review by plafer — APPROVED** [review]
🙏

**Pull request merged**